layouts/partials/topics.html

```diff
@@ -0,0 +1,17 @@
+{{ $topics := slice "covid_19" "antibiotic_resistance" "infectious_diseases" "mpox" }}
+<div class="row row-cols-2 row-cols-lg-3 g-2 g-lg-3">
```

Comment:
```suggestion
<div class="row row-cols-1 row-cols-sm-2 row-cols-lg-3 g-2 g-lg-3">
```
This will make it into single row for smaller displays